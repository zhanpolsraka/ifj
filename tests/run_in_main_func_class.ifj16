//3
class Main{
	static int a = 3;
	static int b = 4;
	//run() is missing!!!
}
