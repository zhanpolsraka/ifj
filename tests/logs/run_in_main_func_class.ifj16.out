
Unknown class begin: expected "class" keyword on line 5

---------------FILENAME: tests/run_in_main_func_class.ifj16---------------
----> parser, class
